src/style.css

```diff
@@ -0,0 +1,39 @@
+body {
+  margin: 0;
+}
+
+.star-container {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  height: 12px;
+  width: 12px;
+  margin: 2px 4px;
+  background-size: cover;
+}
+
+.stars__star:first-child {
+  margin-left: 2px;
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n+2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n+3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 :nth-child(n) {
```

Comment:
the modifier says there should be 5 active stars. now imagine I change my design and there are 7 stars in each row. the design will be broken